### PR TITLE
more documentation

### PR DESCRIPTION
**Pull request by botany-demo**
more documentation

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -43,3 +43,5 @@ update tests for new email sending flow
 use new logo
 
 update auth flow
+
+more documentation
```
Comment (botany-demo) [inline readme.md:47]: any input here @botany-demo @botany-dummy

Comment (botany-lead) [reply readme.md:47]: maybe add more of the documentation?

Comment (botany-demo) [reply readme.md:47]: cool, good idea

Comment (botany-demo) [reply readme.md:47]: ok, added it @botany-lead

Comment (botany-lead) [reply readme.md:47]: perfect, looks good to me

**Review by botany-lead — APPROVED** [review]
much more thorough 👍

**Pull request merged**